PRODUCTIONISATION.md

```diff
@@ -107,32 +106,38 @@ To be applied if infrastructure has been orchestrated and provisioned specifical
 * [ ] 8.1 Sub-organisation created under parent organisation for each customer
 * [ ] 8.2 Monitoring agent be installed on all production infrastructure
 * [ ] 8.3 Monitoring agent could be installed on non-production infrastructure if necessary
-* [ ] 8.4 Monitoring agent hostname configured according to proposed [RFC 008 - Server naming convention][rfc_008]
+* [ ] 8.4 Monitoring agent hostname configured according to proposed [RFC 008 - Server naming convention][rfc_008] (_this doesn't apply to serverless_)
 * 8.5 Monitors configured in infrastructure monitoring service for (warn/alert):
   * [ ] 8.5.1 Inode usage (80% / 90%)
   * [ ] 8.5.2 CPU load (75% / 85% average over 2 minutes)
   * [ ] 8.5.3 Disk usage (80% / 90%)
-  * [ ] 8.5.4 Memory usage (80% / 90% average over 2 minutes)
+  * [ ] 8.5.4 Memory usage (60% / 70% average over 2 minutes)
   * [ ] 8.5.5 Host stopped reporting
   * [ ] 8.5.6 NTP sync
 * 8.6 Monitors configured to alert via:
   * [ ] 8.6.1 Slack channel
   * [ ] 8.6.2 Team email address
 
-### 9. Vulnerability alerts
+### 9. Application performance management
 
-* [ ] 9.1 Configure either VCS hosting service or Continuous Delivery pipeline to detect and alert on vulnerable dependencies
-* [ ] 9.2 Configure notifications to send alerts
+* 9.1 APM service configured to monitor
+  * [ ] 9.1.1 App performance
+  * [ ] 9.1.2 Request time
 
-### 10. Dependency upgrade alerts
+### 10. Vulnerability alerts
 
-* 10.1 [ ] Configure a dependency upgrade notification service to submit pull requests to VCS service repository with alerts
+* [ ] 10.1 Configure either VCS hosting service or Continuous Delivery pipeline to detect and alert on vulnerable dependencies
+* [ ] 10.2 Configure notifications to send alerts
```

Comment:
`10.2` seems redundant, as `10.1` already discusses alerts.

Reply:
Resolved in https://github.com/madetech/productionisation/pull/51/commits/1c577fa41042c5daf4690cbfea6b3f1f8133df52.